### PR TITLE
Merge branch main into 3.0

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
Process:

```sh
$ git checkout 3.0
$ git pull
$ git checkout -b <this-branch>
$ git fetch
$ git merge main
```

Then a seemingly lot of merge conflicts. I ended up regenerating package-lock and thankfully there are not many changes. Just the releases.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
         "./packages/labs/analyzer:build",
         "./packages/labs/cli:build",
         "./packages/labs/compiler:build",
+        "./packages/labs/compiler:build",
```
Comment (augustjk) [inline package.json:59]: this somehow got added again when there's one right above it, causing build breakage.

Comment (AndrewJakubowicz) [reply package.json:59]: hahaha! darn `git merge`! Thank you for catching that!

File: .prettierignore [hunk 2]
```diff
@@ -361,20 +361,20 @@ packages/labs/vue-utils/node_modules/
...
 packages/lit-starter-ts/**/rollup-config.js
 packages/lit-starter-ts/**/custom-elements.json
 packages/lit-starter-ts/**/web-dev-server.config.js
 
+packages/lit-starter-js/node_modules/*
+packages/lit-starter-js/docs/*
+packages/lit-starter-js/docs-src/*
+packages/lit-starter-js/**/rollup-config.js
+packages/lit-starter-js/**/custom-elements.json
+packages/lit-starter-js/**/web-dev-server.config.js
```
Comment (AndrewJakubowicz) [inline .prettierignore:376]: These were moved automatically by husky which was surprising.

File: package.json (hunk 1 above)
Comment (augustjk) [reply package.json]: git-merge

**Review by augustjk — APPROVED** [review]

**Review by rictic — APPROVED** [review]

**Comment by AndrewJakubowicz** [comment]
Is this a "squash and merge" situation, or a "rebase and merge" situation?

**Comment by AndrewJakubowicz** [comment]
This was not merged with GitHub UI – which would have squashed the history.

Instead I checkout out the 3.0 branch.

```sh
git pull
git git merge aug-21-merge-3.0 --ff-only # Ensure this is successful
git push
```

**Pull request merged**